mitodl/mitxpro
# Revert "chore: add support for Heroku-22 stack (#2430)"

This reverts commit 5a4bde552c2bdc89702740eef78146c4012a6012.

#### Pre-Flight checklist


#### What are the relevant tickets?
Related to #2484 

#### What's this PR do?
Reverts the PR that prepared the app for the heroku-22 stack (the PR's most significant change was python 3.7 => 3.9)
Keeps the newer `black` version though to avoid lots of reformatting

#### How should this be manually tested?
Tests should pass, functionality should still work

One caveat on the downgrade - something seems to have changed with how redis stores the wagtail image cache, I had to flush it after downgrading - which might be something to look into further as a possible factor in the performance issue. 

```python
import os
import redis
from urllib.parse import urlparse

r = redis.Redis(host="redis")
r.flushall()
```
